### PR TITLE
Fix update route

### PR DESCRIPTION
**Pull request by QuentinBouygues**
Small fix.

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ app.delete('/lists/:listId', List.delete)
 app.get('/boards/:boardId', Board.getOneById)
 app.get('/users/:user_id/boards', Board.getAllFromUserId)
```
Comment (Porquepix) [inline src/app.ts]: cammel case when possible. user_id => userId. Change it in the route too.

File: src/bl/boardFacade.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ export class BoardFacade {
 
     static async update(boardReceived: Board, boardId: number): Promise<void> {
         try {
-            const board = ParamsExtractor.extract<Board>(['title', 'isPrivate'], boardReceived)
+            const board = ParamsExtractor.extract<Board>(['name', 'isPrivate'], boardReceived)
```
Comment (Porquepix) [inline src/bl/boardFacade.ts]: Use the param extractor v2.

File: src/bl/boardFacade.ts [hunk 3]
```diff
@@ -60,7 +60,7 @@ export class BoardFacade {
...
             const repository = getManager().getRepository(Board)
```
Comment (Porquepix) [inline src/bl/boardFacade.ts]: Manager is useless.

**Review by Porquepix — CHANGES_REQUESTED** [review]
You should follow the List example for naming conventions.

**Comment by QuentinBouygues** [comment]
💇‍♂️ @Porquepix

File: src/bl/boardFacade.ts [hunk 4]
```diff
@@ -94,10 +95,20 @@ export class BoardFacade {
         }
     }*/
 
-    static async create(board: Board): Promise<Board> {
+    static async create(params: {}): Promise<Board> {
         try {
-            let boardToCreate = ParamsExtractor.extract<Board>(['title', 'isPrivate'], board)
-            return getManager().getRepository(Board).create(boardToCreate)
+            const extractor = new ParamsExtractor<Board>(params).permit(['name', 'isPrivate'])
```
Comment (Porquepix) [inline src/bl/boardFacade.ts]: I think name should be required for board ?

**Review by Porquepix — APPROVED** [review]

**Pull request merged**